routers/api/v1/user/star.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2016 The Gogs Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package user
+
+import (
+	api "code.gitea.io/sdk/gitea"
+
+	"code.gitea.io/gitea/modules/context"
+	"code.gitea.io/gitea/models"
+)
+
+// Returns the repos that the user with the specified userID has starred
```

Comment:
While this function isn't exported (therefore doesn't need a lint-comment), it would be nice to follow conventions everywhere :)